app/locales/es/translations.js

```diff
@@ -481,173 +590,414 @@ export default {
       reportType: 'Tipo de reporte',
-      stockTransferDetail: 'Detalle de traslado de stock',
-      stockTransferSum: 'Resumen traslado de stock',
-      stockUsageDetail: 'Detalle de uso de stock',
-      stockUsageSum: 'Resumen uso de stock'
+      stockTransferDetail: 'Detalle de traslados',
+      stockTransferSum: 'Resumen traslados',
+      stockUsageDetail: 'Detalle de Uso',
+      stockUsageSum: 'Resumen de Uso'
     },
     titles: {
-      addPurchase: 'Adicionar compra',
-      addRequest: 'Nuevo pedido',
+      addPurchase: 'Agregar Compra',
+      addRequest: 'Nuevo Pedido',
       adjustment: 'Ajuste',
-      editPurchase: 'Editar compra',
-      editRequest: 'Editar pedido',
-      inventoryItem: 'Nuevo elemento de inventario',
-      inventoryReport: 'Reporte de inventario',
-      purchaseSaved: 'Compras de inventario guardadas',
-      quickAddTitle: 'Nuevo elemento de inventario',
-      removeItem: 'Remover elemento',
-      requestFulfilled: 'Pedido cumplido',
-      requestUpdated: 'Pedido actualizado',
-      transfer: 'Trasladar elementos',
-      warning: 'Peligro!!!!!'
+      editPurchase: 'Editar Compra',
+      editRequest: 'Editar Pedido',
+      inventoryItem: 'Nuevo Elemento de Inventario',
+      inventoryReport: 'Reporte de Inventario',
+      purchaseSaved: 'Compras de Inventario Guardadas',
+      quickAddTitle: 'Nuevo Eelemento de Inventario',
+      removeItem: 'Remover Elemento',
+      requestFulfilled: 'Pedido Cumplido',
+      requestUpdated: 'Pedido Actualizado',
+      transfer: 'Transferencia de Elementos',
+      warning: '¡Advertencia!'
     }
   },
   imaging: {
     pageTitle: 'Pedido de radriografia',
     sectionTitle: 'Radiografias',
     buttons: {
-      newButton: '+ new radiografia'
+      newButton: '+ new radiografía'
     },
     labels: {
-      radiologist: 'Radiologo',
-      addNewVisit: '--Adicionar nueva visita--'
+      radiologist: 'Radiólogo',
+      addNewVisit: '--Agregar nueva visita--'
     },
     messages: {
-      noCompleted: 'No de items encontrados.'
+      delete: 'Esta seguro de que desea eliminar esta solicitud de radiografía ?',
+      noCompleted: 'No se encontrarón los elementos terminados.'
     },
     titles: {
-      completedImaging: 'Radiografia completada',
-      editTitle: 'Pedido de edicion de radiografia',
-      newTitle: 'Nuevo pedido de radiografia'
+      completedImaging: 'Radiografía completada',
+      editTitle: 'Pedido de edicion de radiografía',
+      newTitle: 'Nuevo pedido de radiografía'
     },
     alerts: {
-      completedTitle: 'Pedido de radiografia completado',
-      completedMessage: 'El pedido de radiografia ha sido completado.',
-      savedTitle: 'Pedido de radiografia salvado',
-      savedMessage: 'El pedido de radiografia ha sido guardado.'
+      completedTitle: 'Pedido de radiografía completado',
+      completedMessage: 'El pedido de radiografía ha sido completado.',
+      savedTitle: 'Pedido de radiografía salvado',
```

Comment:
Did you mean `guardado`?

`Salvado` doesn't mean what you think it does:
http://dle.rae.es/?id=X7tj9JF